packages/rx_bloc_cli/mason_templates/bricks/lib_router/__brick__/lib/lib_router/router.dart

```diff
@@ -2,6 +2,8 @@
 
 import 'dart:async';
 
+{{#enable_auth_matrix}}
+import 'package:equatable/equatable.dart';{{/enable_auth_matrix}}
```

Comment:
What's the reason for this import?

Reply:
The auth matrix-related routes need to be equatable so that the mockito argument matching mechanism can work as expected.